admin-dev/themes/default/template/helpers/form/form.tpl

```diff
@@ -770,7 +770,7 @@
 												type="text"
 												data-hex="true"
 												{if isset($input.class)} class="{$input.class}"
-												{else} class="datetimepicker"{/if}
+												{else} class="datetimepicker" autocomplete="off"{/if}
```

Comment:
Need to be out of the if/else class related.

Reply:
No, if a class is set, the datepicker is not launched.

Reply:
You could also have the class set as "datepicker myclassname".